LottieDropper/Dropbox/DropboxBrowserViewModel.m

```diff
@@ -177,37 +177,26 @@ -(void) setupFileDetailsFromLocalFilenames {
 // 4. Add empty files for all dropbox files
 - (void) mergeDropboxFileListWithFileDetails {
     
-    // 1. Alle files op dropbox aan een detail linken
-    
-    [self.dropboxFileCache enumerateObjectsUsingBlock:^(DBFILESMetadata * file, NSUInteger idx, BOOL *stop) {
```

Comment:
Dit is dus wel nodig maar kan weg gewerkt worden door detail een DBFilesMetadata aan te laten maken vanaf een file naam.